### ISSUE TITLE
loss explode

### ISSUE THREAD

**Issue by navy63** [TragedyOwl/EDSR-Tensorflow#1]
The original version will explode  20,000 times in training. I don't know whether your version will explode or not.

**Comment by navy63** [comment]
This code is useless. The loss value will explode.
